src/analysis/clustering.jl

```diff
@@ -133,3 +133,51 @@ function time_series_clustering(result_set::ResultSet, data::AbstractMatrix{T},
     # Assign cluster 0 for filtered sites to use same indexing as the input data
     return [site != 0 ? popfirst!(clusters) : 0 for site in non_null_sites]
 end
+
+"""
+    target_clusters(clusters::Vector{T}, outcomes::AbstractMatrix{F}; metric=temporal_variability, size_limit=0.01) where {T<:Int64,F<:Real}
+
+Categorize all clusters into target or non-target according to some metric.
+
+# Arguments
+- `clusters` : Vector with outcome cluster indexes
+- `outcomes` : AbstractMatrix of scenario outcomes
+- `metric` : Metric used to aggregate outcomes for each cluster
+- `size_limit` : This function will iteratively merge the best cluster with the second best
+    if the fraction of scenarios inside it is below `size_limit`
+
+# Returns
+Vector containing 1's for target and 0's for non-target clusters
+"""
+function target_clusters(clusters::Vector{T}, outcomes::AbstractMatrix{F};
+    metric=temporal_variability, size_limit=0.01)::Vector{T} where {T<:Int64,F<:Real}
+
+    # Compute statistic for each cluster
+    clusters_statistics::Vector{Float64} = []
+    for cluster in unique(clusters)
+        normalized_outcomed = outcomes[:, clusters.==cluster] ./ maximum(outcomes)
+        statistic = median(metric(normalized_outcomed))
+        push!(clusters_statistics, statistic)
+    end
+
+    target_index = argmax(clusters_statistics)
+    target_indexes = [target_index]
+
+    # Merge target cluster if is above 1% of size

```

Comment:
```suggestion
    # Merge target cluster if it is above 1% of size

```

Is this meant to be "below 1%"?

Reply:
Yes.